assets/sass/components/analytics-4/audience-segmentation/_googlesitekit-audience-selection-panel.scss

```
@@ -207,6 +207,50 @@
 				width: initial;
 			}
 		}
+
+		.googlesitekit-audience-creation-error-notice {
+			align-items: center;
+			background-color: $c-content-on-primary;
+			border: 2px solid $c-utility-error;
+			border-radius: $br-sm;
```

According to Figma, the border-radius is 8px.

```suggestion
			border-radius: $br-xs;
```

---

Updated.